src/domain/search/results/__tests__/__snapshots__/SearchListResults.test.js.snap

```diff
@@ -35,7 +35,7 @@ exports[`SearchListResults renders correctly 1`] = `
       className="app-SearchListResults__results"
     >
       <InjectIntl(Connect(InjectT(ResourceCard)))
-        date="2019-08-05"
+        date="2019-08-06"
```

Comment:
I think it's better to set this value to fixed date (@midori0507 has implemented a util function for that purpose), otherwise, this snapshot will fail tomorrow.

Reply:
This should be fixed now that I merged develop back into this branch.